volunteers.js

```diff
@@ -79,13 +79,13 @@ module.exports = async function startVolunteers() {
           break;
         }
 
-        if (Math.abs(hours)> 24) {
+        if (Math.abs(hours) > 24) {
           message.reply('I can only add or remove a maximum of 24 hours! ' +
             'Please try again?\n');
           break;
         } 
         
-        if (hours === 0) {
+        if (hours == 0) {
```

Comment:
I would much prefer it to explicitly convert hours to a number and compare with a strict triple-equals sign, rather than relying on double-equals because double-equals increase the chances of the reader to misinterpret. (It makes the design intention of the code ambiguous - e.g. is it a mistake or is it intended?).